README.md

```diff
@@ -31,6 +31,22 @@ Mr Mustard supports:
 - Plug-and-play backends (TensorFlow as default)
 - An abstraction layer `XPTensor` for seamless symplectic algebra (experimental)
 
+# Increased numerical stability using Julia [optional]
+
+Converting phase space objects to Fock space can be numerically unstable due to accumulating floating point errors.
+To resolve this, the conversion can be performed with extended-precision arithmetic. To use this feature,
+an installation of [Julia](https://github.com/JuliaLang/juliaup#installation) is required (version 1.9.3).
```

Comment:
Is 1.9.3 the only one that's gonna work? 
What do people need to do if they have >1.9.3? Downgrade?

Reply:
I would think everything keeps working when newer versions come out, but I haven't checked older versions. Should I just write >1.9.3? Or should I check which ones currently work?

Reply:
1.9.3 is the current stable release. Should I already write >= or do we wait until newer versions come out?

Reply:
Great, thanks!